source/extensions/filters/network/dubbo_proxy/router/route_matcher.cc

```diff
@@ -184,7 +184,12 @@ RouteConstSharedPtr SingleRouteMatcherImpl::route(const MessageMetadata& metadat
   ASSERT(metadata.hasInvocationInfo());
   const auto& invocation = metadata.invocation_info();
 
-  if (service_name_ == invocation.service_name() &&
+  bool interface_check = true;
```

Comment:
Here we should use StringMatcher more appropriate

Reply:
I have used StringMatcher instead，however because I modefied the BUILD file，bot check failed